tensorrt_llm/media/encoding.py

```
@@ -141,45 +141,47 @@ def encode_video(
+                    "-s", f"{width}x{height}", "-r", str(frame_rate), "-i", "-",
+                    *audio_input_args, *codec_args, *audio_output_args, output_path,
+                ]
+                try:
+                    proc = subprocess.Popen(
+                        cmd, stdin=subprocess.PIPE,
+                        stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                    )
+                    _, err = proc.communicate(input=raw_frames)
+                    return proc.returncode, err
+                except FileNotFoundError:
+                    raise RuntimeError("ffmpeg not found. Install ffmpeg for video encoding.")
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)